modules/installation-creating-aws-bootstrap.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-bootstrap-{context}"]
+= Creating the bootstrap node in AWS
+
+You must create the bootstrap node in Amazon Web Services (AWS) to use during
+{product-title} cluster initialization. The easiest way to create this node is
+to modify the provided CloudFormation template.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your bootstrap
+node, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and compute roles.
+
+.Procedure
+
+. Provide a location to serve the `bootstrap.ign` Ignition config file to your
+cluster. This file is located in your installation directory. One way to do this
+is to create an S3 bucket in your cluster's region and upload the Ingition
+config file to it.
++
+[IMPORTANT]
+====
+The provided CloudFormation Template assumes that the
+Ignition config files for your cluster are served from an S3 bucket. If you
+choose to serve the files from another location, you must modify the templates.
+====
++
+[NOTE]
+====
+The bootstrap Ignition config file does contain secrets, like X.509 keys. If
+uploading the Ignition config file to an S3 bucket does not provide adequate
+security, you can enable an S3 bucket policy to allow only certain users,
+such as the OpenShift IAM user, to access objects that the bucket contains.
```

Comment:
"If uploading to an S3 bucket is not secure enough, tighten down your S3 bucket" doesn't feel quite right.  How about:

> The bootstrap Ignition config file does contain secrets, like X.509 keys.  The steps below provide basic security for the S3 bucket.  To provide additional security, you can enable an S3 bucket policy to allow only certain users, such as the OpenShift IAM user, to access objects that the bucket contains.  And you can avoid S3 entirely and serve your bootstrap Ignition config from any address reachable from the bootstrap machine.

I'm also fine not enumerating possible S3 tightenings, and saying something like:

> The bootstrap Ignition config file does contain secrets, like X.509 keys.  The steps below provide basic security for the S3 bucket.  If you have security concerns with these steps, you can configure a more-secure S3 bucket or avoid S3 entirely and serve your bootstrap Ignition config from any address reachable from the bootstrap machine.

Reply:
@wking, please check again?

Reply:
f674f0e20fd6 looks good to me.